Library/Formula/docker-machine-driver-xhyve.rb

```diff
@@ -0,0 +1,49 @@
+require "language/go"
+
+class DockerMachineDriverXhyve < Formula
+  desc "Docker Machine driver for xhyve"
+  homepage "https://github.com/zchee/docker-machine-driver-xhyve"
+  url "https://github.com/zchee/docker-machine-driver-xhyve/releases/download/v0.2.1/docker-machine-driver-xhyve"
```

Comment:
@xu-cheng When use Go vendoring (`vendor`), `archive/v*.*.*.tar.gz` is too large.
There is a case to be timeout when downloading use `curl`.

Nevertheless it must be the source tarball?